pkg/parsers/kernel/uname_linux.go

```diff
@@ -4,6 +4,7 @@ import (
 	"syscall"
 )
 
+// Utsname utsname structure is used to hold information returned by the uname function
 type Utsname syscall.Utsname
```

Comment:
Yeah, I *aligned* to the ``syscall.Utsname`` so I did not use ``UTS…``.
But ``type Utsname`` is not used anywhere in docker codebase, the ``uname()`` function returns the ``syscall.Utsname``, so it feels like this type doesn't even need to be *defined*.